xfrd.h

```diff
@@ -239,9 +239,10 @@ enum xfrd_packet_result {
    And it should be below FD_SETSIZE, to be able to select() on replies.
    Note that also some sockets are used for writing the ixfr.db, xfrd.state
    files and for the pipes to the main parent process.
+
+   For xfrd_tcp_max 128 can be the number of TCP AXFR/IXFR concurrent
```

Comment:
This fixed setting changed to a configurable setting, but the default value changed as well.
```suggestion
   For xfrd_tcp_max , 32 is the default number of TCP AXFR/IXFR concurrent
```

Reply:
Fixed this in https://github.com/NLnetLabs/nsd/pull/193/commits/ed8117cc2c067fb5b4a6db08e819503c3842f7bf
So I used 32 because the 64K preallocated buffer is large, and with 32 that is a much smaller footprint than with 128. But it is still not that big with 128, so I reverted that back to a default of 128. So there is no change of default in the PR.